ormconfig.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  type: 'mysql',
+  host: process.env.JAWSDB_HOST || 'localhost',
+  port: 3306,
+  username: process.env.JAWSDB_USERNAME || 'newuser',
+  password: process.env.JAWSDB_PASSWORD || 'password',
+  database: process.env.JAWSDB_DATABASE || 'bulletinboard2',
```

Comment:
These default values don't make sense now because they probably make sense for one user only :) So you have to find a way that also locally developing you will use values from `process.env `